includes/admin/importers/class-wc-product-csv-importer-controller.php

```diff
@@ -295,7 +295,7 @@ public function handle_upload() {
 				'test_form' => false,
 				'mimes'     => $valid_filetypes,
 			);
-			$import    = wp_unslash( $_FILES['import'] ); // WPCS: sanitization ok.
+			$import    = wc_clean( $_FILES['import'] ); // WPCS: sanitization ok.
```

Comment:
I don't think we need to call `wc_clean()` or `wp_unslash()` here. `wp_handle_upload()` since handles sanitizing file names:

https://github.com/WordPress/WordPress/blob/b4320bc/wp-admin/includes/file.php#L827
https://github.com/WordPress/WordPress/blob/fdd5b8d/wp-includes/functions.php#L2137

An example of an instance where WP core passes a file from `$_FILES` directly to `wp_handle_upload()`:

https://github.com/WordPress/WordPress/blob/fdd5b8dacdf5f3e107747e9870c2f2db3f3481b1/wp-admin/custom-background.php#L488

Am I missing something?

`wp_unslash()` was probably added by mistake during the PHPCS fixes effort mislead by this PHPCS message:

"Missing wp_unslash() before sanitization."

Reply:
Just to elaborate a little bit more, I fear that `wc_clean()` might introduce other bugs similar to the one that `wp_unslash()` introduced while sanitizing `$_FILES['import']` so I suggest we keep it only if necessary which I don't think it is.

Reply:
What I suggest is that you remove the line that declares `$import` and change the next line to:

```
$upload    = wp_handle_upload( $_FILES['import'], $overrides ); // phpcs:ignore WordPress.VIP.ValidatedSanitizedInput.MissingUnslash
```